Week1/homework/js2-exercises/1-books.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Book list</title>
+</head>
+
+<body>
+  <h1>My book list</h1>
+
+  <script>
+    const books = [
+    {
+    title : 'An Artist of the Floating World',
+    author : 'Kazuo Ishiguro',
+    alreadyRead :false  
+    },
+    {
+    title : 'Underworld',
+    author : 'Don DeLillo',
+    alreadyRead :true   
+    }
+];
+
+    const ul = document.createElement('ul');
+    // 1
+    for (let obj of books) {
+// 2
+      const p = document.createElement('p');
+      p.innerText = `${obj.title} was written by ${obj.author.toUpperCase()}`;
+// 5
+      
+      if (obj.alreadyRead === true) {
+        p.style.color = 'green';
+      } else {
+        p.style.color = 'red';
+      }
+    
+// 3
+      const li = document.createElement('li');
+      li.appendChild(p);
+      ul.appendChild(li);
+      document.body.appendChild(ul);
+    }
+    // 4
+    let img1 = document.createElement('img');
+    let img2 = document.createElement('img');
+    img1.src = 'https://upload.wikimedia.org/wikipedia/en/thumb/e/e1/ArtistOfTheFloatingWorld.jpg/220px-ArtistOfTheFloatingWorld.jpg';
+    img2.src = 'http://xoxoafterdark.com/wp-content/uploads/2014/10/Delillo-Underworld.jpg';
+    let lis = document.querySelectorAll('li');
```

Comment:
Think about why would you query the list items you already have them inside of the for of loop right?